### PR TITLE
Replace stamp bash script with node script

### PR DESCRIPTION
**Pull request by fredericbonnet**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no API changes)
- [X] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #366

## What is the new behavior?

The behavior of the NPM script `stamp` is unchanged, however it no longer depends on external dependencies such as Bash or Perl. The original script has been rewritten as a cross-platform Node JS script. Dependencies have been added to the `devDependencies` section in `package.json`.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information

Happy Hacktoberfest!

### CODE REVIEW

**Comment by Jefiozie** [comment]
Hi @fredericbonnet, thank you for your PR. I will put it on my list to review it when I have some time. Happy hacktoberfest

**Comment by Jefiozie** [comment]
I did some local tests and everything works as expected, thanks for the PR. As this is an NX workspace we are trying to use the advised structures from NRWL. One of the things is that scripts that are being used within the repo should live in the tools folder [see the docs here](https://nx.dev/latest/angular/workspace/workspace-overview#files). Could you replace the file with the tools directory?

**Comment by fredericbonnet** [comment]
Sure, done!

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "update": "ng update @nrwl/workspace",
     "workspace-schematic": "nx workspace-schematic",
     "dep-graph": "nx dep-graph",
-    "stamp": "nx run xlayers:stamp",
+    "stamp": "node scripts/stamp.js",
```
Comment (Jefiozie) [inline package.json:28]:
```suggestion
    "stamp": "node tools/stamp.js",
```

Comment (Jefiozie) [reply package.json:28]: Could you change this to the suggested change?

**Review by Jefiozie — COMMENTED** [review]
If the small comment is fixed it LGTM. 

cc: @manekinekko do you agree on this change?

File: package.json (hunk 1 above)
Comment (fredericbonnet) [reply package.json]: Oops sorry for the mistake, I shouldn't push code late on a sunday night! :) Fixed.

**Review by Jefiozie — APPROVED** [review]
LGTM

**Review by manekinekko — APPROVED** [review]

**Comment by manekinekko** [comment]
Thanks a lot @fredericbonnet for your contribution. We really appreciate it 🎉

**Pull request merged**